.tekton/tasks/verify-smoke-e2e.yaml

```diff
@@ -0,0 +1,87 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: stage-verify-e2e-test
+  namespace: rhtap-staging-testing
+spec:
+  description: >-
+    This task is for running stage e2e test on RHTAP staging cluster
+  params:
+  - name: PLATFORM
+    description: which environment the test is running on.
+    default: "stage"
+  - name: TASK_RETRIES
+    description: number of times the task will be rerun
+    default: "0"
+  workspaces:
+    - name: stage-e2e-git
+  steps:
+    - name: run-stage-verify-e2e-test
+      image: registry.access.redhat.com/ubi9/go-toolset:1.20.10-2.1699551725
+      workingDir: $(workspaces.stage-e2e-git.path)
+      script: |
+        #!/usr/bin/env bash
+        set -eufx
+
+        mkdir -p ~/.kube
+        cat <<EOF > ~/.kube/config
+        apiVersion: v1
+        clusters:
+        - cluster:
+            server: YOUR_KUBE_API_SERVER_URL
+          name: YOUR_CLUSTER_NAME
+        contexts:
+        - context:
+            cluster: YOUR_CLUSTER_NAME
+            user: YOUR_USER_NAME
+          name: YOUR_CONTEXT_NAME
+        current-context: YOUR_CONTEXT_NAME
+        kind: Config
+        preferences: {}
+        users:
+        - name: YOUR_USER_NAME
+          user:
+            token: YOUR_KUBE_API_TOKEN
+        EOF
+        make build
+        E2E_ARGS_EXEC="--ginkgo.label-filter $(params.PLATFORM) --ginkgo.vv" make run
```

Comment:
where is this implementation done? I don't see any label platform in e2e...

Reply:
here is the value of PLATFORM from the params https://github.com/redhat-appstudio/infra-deployments/pull/2383/files#diff-67ea61e64b6eefdd1070b54cc9c36ace29982f53260d6ed09ae7e583b7a347a9R10-R12

Reply:
yes, but i dont see this label in e2e

Reply:
It's still in [Meer's repo](https://github.com/sawood14012/e2e-tests/blob/verify-stage/tests/stage/verifystage.go#L17) and not yet merged to the main repo. @sawood14012 do you have any timeline for that.

Reply:
This should not be the blocker for this PR IMO

Reply:
@flacatus i will add these changes today